test/README.md

```diff
@@ -0,0 +1,456 @@
+# Cluster Kube API Server Operator Tests Extension
```

Comment:
I appreciate the README file, but I think it's a bit too verbose. For now, we could start with some basic information. We could also add an additional section to the existing README.